test/libgit2-online.jl

```diff
@@ -37,12 +36,13 @@ mktempdir() do dir
 
         #TODO: remove or condition on libgit2 features this test when ssh protocol will be supported
         #@testset "with 'ssh' protocol (by default is not supported)" begin
+            repo_path = joinpath(dir, "Example3")
+            repo = LibGit2.clone(repo_url_git, repo_path)
             try
-                repo_path = joinpath(dir, "Example3")
-                @test_throws LibGit2.Error.GitError LibGit2.clone(ssh_prefix*repo_url, repo_path)
-            catch ex
-                # but we cloned succesfully, so check that repo was created
-                ex.fail == 1 && @test isdir(joinpath(path, ".git"))
+                @test isdir(repo_path)
+                @test isdir(joinpath(repo_path, ".git"))
+            finally
+                finalize(repo)
```

Comment:
Turns out the test was only successful on my local machine since I had SSH keys setup. I'll remove the test as in its current state it'll probably pass on machines with GitHub SSH keys but fail on everything else including the CI.